pkg/skaffold/debug/transform_python.go

```diff
@@ -103,11 +114,17 @@ func (t pythonTransformer) Apply(container *v1.Container, config imageConfigurat
 		pyUserBase = pyUserBase + ":" + existing
 	}
 	container.Env = setEnvVar(container.Env, "PYTHONUSERBASE", pyUserBase)
```

Comment:
This code to set `PYTHONUSERBASE` can now be removed as the right values are set by the launcher.